en/Plugins/Bookmarks.md

```diff
@@ -0,0 +1,53 @@
+---
+alias: Plugins/Starred
+---
+
+You can bookmark files and organize them into bookmark groups to find them easier.
+
+You can find your bookmarked files by clicking the **Bookmarks** (bookmark icon) tab in the left sidebar.
+
+Or, open the Bookmarks tab by running **Bookmarks: Show bookmarks** from the [[Command palette]].
+
+Open a bookmark by selecting it in the Bookmarks tab.
+
+You can reorder bookmarks by selecting and dragging them.
+
+## Add a bookmark
+
+To bookma rk a note:
+
+**Bookmarks tab**:
+
+- Select **Bookmark the active tab** (bookmark with plus icon) at the top of the Bookmarks tab.
+
+**File explorer:**
+
+- Right-click a file in [[File explorer]] and click **Bookmark**.
+
+**Command palette**:
+
+1. Press `Ctrl+P` (or `Cmd+P` on macOS) to open the [[Command palette]].
+2. Search for the **Bookmark** command.
+3. Press `Enter` with the command selected to toggle the active note.
```

Comment:
This doesn't toggle the active note. This opens the bookmark modal which gives you the option to add a title, select a bookmark group or (if the bookmark already exists) the option to remove the bookmark.